ebill_paynet/models/paynet_invoice_message.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import os
+from datetime import datetime
+
+# Needs Jinja 2.10
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+from zeep.exceptions import Fault
+
+from odoo import fields, models
+from odoo.modules.module import get_module_root
+
+from ..components.api import PayNetDWS
+
+MODULE_PATH = get_module_root(os.path.dirname(__file__))
+INVOICE_TEMPLATE_2013 = "invoice-2013A.xml"
+INVOICE_TEMPLATE_2003 = "invoice-2003A.xml"
```

Comment:
As they are Jinja templates:
```suggestion
INVOICE_TEMPLATE_2013 = "invoice-2013A.xml.j2"
INVOICE_TEMPLATE_2003 = "invoice-2003A.xml.j2"
```
(files to rename accordingly)